design-reviews/trade-studies/template.md

```diff
@@ -0,0 +1,83 @@
+# Trade Study Template
```

Comment:
Love it. I am doing this same activity this week to evaluate Blob storage vs ADLS. Will definitely make use of this on the next "trade-study" I do.

Reply:
Great! Hopefully you find it helpful and we can continue to iterate on this idea